### ISSUE TITLE
Why ts error in .vue file can not be displayed  in Vite HMR overlay 

### ISSUE THREAD

**Issue by duola8789** [fi3ework/vite-plugin-checker#56]
My IDE is Webstorm, I can get ts error in Typescript terminal, but it can not be dispalyed  in Vite HMR overlay

```ts
export default defineComponent({
    name: 'App',
    components: {
        HelloWorld
    },
    setup() {
        const a = ref<string>(22223);
        a.value = 22212;
        return {a};
    }
});
```
![image](https://user-images.githubusercontent.com/17665968/126982728-8b140efd-7a02-4816-bb68-631cd715b834.png)

And Vite HMR overlay also doesn't show the error

![image](https://user-images.githubusercontent.com/17665968/126982774-16d1d7fc-70c6-4d0e-aa15-d2a16fdca060.png)

is is possible to show the error on broswer just like Vue CLI?

![image](https://user-images.githubusercontent.com/17665968/126982870-05b667a2-8527-4f4f-b5de-eb0a9ff7042b.png)

**Comment by fi3ework** [comment]
Could you provide a minimal way to reproduce this project?

**Comment by duola8789** [comment]
> Could you provide a minimal way to reproduce this project?

https://github.com/duola8789/vite-ts

You could use this repository to reproduce, and the ts type error could be found at app.vue
thanks

**Comment by fi3ework** [comment]
@duola8789 Thanks. I reproduce the demo. It can be fixed with a little tweak to VLS initial configuration (for VTI it works). But it needs a little more work to fix in this plugin. I'll fix it these days.

**Comment by duola8789** [comment]
@fi3ework thanks
